components/GeoApi.js

```diff
@@ -127,13 +154,12 @@ function GeoApi(props) {
     fire.firestore().collection(userData.additionalUserInfo.profile.id).doc(location).delete();
     const newList = locationsGrids.filter((item) => item.id !== location);
     setLocationsGrids(newList);
-    console.log(newList);
   };
 
   return (
     <div>
-      <form onSubmit={handleSubmit}>
-        <input type="text" onChange={handleChange} />
+      <form autoComplete="off" onSubmit={handleSubmit}>
```

Comment:
I am unfamiliar with what the autoComplete attribute does here, could you explain it to me?